torchaudio/__init__.py

```diff
@@ -30,3 +32,27 @@ def load(filename, out=None):
     func(bytes(filename), out, sample_rate_p)
     sample_rate = sample_rate_p[0]
     return out, sample_rate
+
+
+def save(filepath, src, sample_rate):
+    assert torch.is_tensor(src)
+    assert not src.is_cuda
```

Comment:
It's better to use ifs and raise proper errors. You can use assertions to check invariants, not arguments